cli/Cargo.toml

```diff
@@ -101,4 +101,4 @@ nix = "0.20.0"
 OriginalFilename = "deno.exe"
 LegalCopyright = "© Deno contributors & Deno Land Inc. MIT licensed."
 ProductName = "Deno"
-FileDescription = "A secure runtime for JavaScript and TypeScript."
+FileDescription = "Deno: A secure runtime for JavaScript and TypeScript"
```

Comment:
FWIW, main website says "A secure runtime for JavaScript and TypeScript." while the repo description says "A secure JavaScript and TypeScript runtime."